client/src/Components/CommonComponents/Header/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+import Logo from './Logo.jpg';
+
+const Contianer = styled.div`
```

Comment:
Container?

Reply:
@RamyAlshurafa I see you've fixed this but now it's "container" as it's a component we should probably capitalise the c so its Container?